target/linux/realtek/dts-5.10/rtl8393_panasonic_m48eg-pn28480k.dts

```diff
@@ -0,0 +1,352 @@
+	 * and they are connected to the RTL8218FB. Currently, there is
+	 * no support for the chip and only TP ports work by the RTL8218B
+	 * support.
+	 */
+	sfp0: sfp-p45 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c0>;
+		tx-fault-gpio = <&gpio1 0 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 1 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 2 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 3 GPIO_ACTIVE_HIGH>;
+	};
+
+	sfp1: sfp-p46 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c1>;
+		tx-fault-gpio = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 6 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+	};
+
+	sfp2: sfp-p47 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c2>;
+		tx-fault-gpio = <&gpio1 8 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 9 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 10 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 11 GPIO_ACTIVE_HIGH>;
+	};
+
+	sfp3: sfp-p48 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c3>;
+		tx-fault-gpio = <&gpio1 12 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 13 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 14 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 15 GPIO_ACTIVE_HIGH>;
+	};
+
+	thermal-zones {
+		cpu-thermal {
+			polling-delay-passive = <1000>;
+			polling-delay = <2000>;
+
+			thermal-sensors = <&tsens0>;
+
+			trips {
+				cpu_alert: trip-point {
+					temperature = <45000>;
+					hysteresis = <4000>;
+					type = "active";
+				};
+
+				cpu_crit {
+					temperature = <55000>;
+					hysteresis = <1000>;
+					type = "critical";
+				};
+			};
+
+			cooling-maps {
+				map {
+					trip = <&cpu_alert>;
+					cooling-device = <&fan 0 1>;
+				};
+			};
+		};
+
+		/*
+		 * Official Spcifications about Temperature (System)
+		 *
+		 * Fan Speed setting:
+		 * - Low : 0-40 celsius
+		 * - High: 0-50 celsius
+		 */
+		sys-thermal {
+			polling-delay-passive = <1000>;
+			polling-delay = <2000>;
+
+			thermal-sensors = <&tsens1>;
+
+			trips {
+				sys_alert: trip-point {
+					temperature = <40000>;
+					hysteresis = <4000>;
+					type = "active";
+				};
+
+				sys_crit {
+					temperature = <50000>;
+					hysteresis = <2000>;
+					type = "critical";
+				};
+			};
+
+			cooling-maps {
+				map {
+					trip = <&sys_alert>;
+					cooling-device = <&fan 0 1>;
+				};
+			};
+		};
+	};
+};
+
+&leds {
+	led_status_eco_amber: led-5 {
+		label = "amber:status_eco";
+		gpios = <&gpio2 1 GPIO_ACTIVE_HIGH>;
+		color = <LED_COLOR_ID_AMBER>;
+		function = LED_FUNCTION_STATUS;
+		function-enumerator = <1>;
+	};
+
+	led_status_eco_green: led-6 {
+		label = "green:status_eco";
+		gpios = <&gpio2 2 GPIO_ACTIVE_HIGH>;
+		color = <LED_COLOR_ID_GREEN>;
+		function = LED_FUNCTION_STATUS;
+		function-enumerator = <2>;
+	};
+};
+
+&i2c_gpio_0 {
+	scl-gpios = <&gpio0 16 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio0 17 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+
+	/* Microchip TCN75A (for SoC) */
+	tsens0: sensor@48 {
```

Comment:
Maybe change the label to something along the lines of "tsens_soc". Then this would be more descriptive, and you don't really need the comment anymore, unless you want to give the reader an idea of where on the board this sensor is.

Reply:
Sounds great, I'll update the label. However, the lm75 driver has no support for TCN75'A' variant and I would like to keep the comment at the moment.

Reply:
I had to look it up, but the difference is apparently that the TCN75A has 12-bit measurement resolution, but the TCN75 only has 9-bit. Then it's OK to keep this comment. Could you add this info to the commit message too?

Reply:
> Could you add this info to the commit message too?

OK, I'll do.